community/modules/compute/htcondor-execute-point/files/htcondor_configure_autoscaler.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: Configure HTCondor Autoscaler
+  hosts: all
+  vars:
+    python: /usr/local/htcondor/bin/python3
```

Comment:
There is a separate runner to install this particular virtual environment. The intended behavior is for that virtual environment to be the only way to run the autoscaler.

There might be a way to fail if the interpreter doesn't exist. It may not be obvious to you, but this is a default value which can be overridden with the `-e` flag to `ansible-playbook`.

Reply:
[Virtual environment creation runner](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/develop/community/modules/scripts/htcondor-install/files/install-htcondor-autoscaler-deps.yml)

Reply:
SG. I thought as much and was looking for where the env was set up but couldn't find it.

nit: I am not sure what etiquette is on comments in ansible. But I would consider adding a comment before the var that says: _# python env set up in hpc-toolkit/community/modules/scripts/htcondor-install/files/install-htcondor-autoscaler-deps.yml_, or something of that nature.

Reply:
Done on `HEAD` of PR.